server/db_ql.go

```diff
@@ -279,6 +320,28 @@ func (qc *QlCache) UpdateFixity(item string, status string, notes string) error
 	return err
 }
 
+//
+func (qc *QlCache) DeleteFixity(id string) error {
+        const query = `
+                DELETE FROM fixity
+                WHERE id() == ?1 and status == "scheduled"`
+
+	intId, _ := strconv.Atoi(id)
+
+        result, err := performExec(qc.db, query, intId)
+        if err != nil {
+                return err
+        }
+        nrows, err := result.RowsAffected()
```

Comment:
I don't know if nrows is worth it. The MySQL adapter doesn't use nrows, for instance.